lib/onc_certification_g10_test_kit/short_id_map.yml

```diff
@@ -0,0 +1,1522 @@
+: 4.15.05
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_weight_for_height-us_core_v311_pediatric_weight_for_height_validation_test
+: 4.15.06
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_weight_for_height-us_core_v311_pediatric_weight_for_height_must_support_test
+: 4.15.07
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_weight_for_height-us_core_v311_pediatric_weight_for_height_reference_resolution_test
+: 4.15.08
+g10_certification-g10_single_patient_api-us_core_v311_observation_lab: '4.16'
+g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_patient_category_search_test: 4.16.01
+? g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_patient_category_date_search_test
+: 4.16.02
+g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_patient_code_search_test: 4.16.03
+g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_read_test: 4.16.04
+? g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_provenance_revinclude_search_test
+: 4.16.05
+g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_validation_test: 4.16.06
+g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_must_support_test: 4.16.07
+g10_certification-g10_single_patient_api-us_core_v311_observation_lab-us_core_v311_observation_lab_reference_resolution_test: 4.16.08
+g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age: '4.17'
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_patient_code_search_test
+: 4.17.01
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_patient_category_date_search_test
+: 4.17.02
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_patient_category_search_test
+: 4.17.03
+g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_read_test: 4.17.04
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_provenance_revinclude_search_test
+: 4.17.05
+g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_validation_test: 4.17.06
+g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_must_support_test: 4.17.07
+? g10_certification-g10_single_patient_api-us_core_v311_pediatric_bmi_for_age-us_core_v311_pediatric_bmi_for_age_reference_resolution_test
```

Comment:
Why the question mark in front of some of the lines?  I figured maybe it had to do with some tests be suppressed because they are optional, but that doesn't look like it?

Reply:
I'd never seen that either before this. Apparently it's a way in yaml to have a complex key. You could use it to have something like an array as a key. I think it switches to that format when the keys reach a certain length. I don't know the particulars about why that means they need the special format keys.